libqtile/backend/x11/xcbq.py

```diff
@@ -927,6 +927,10 @@ def fixup_focus(self):
                 xcffib.xproto.Time.CurrentTime,
             )
 
+    def query_tree(self):
+        q = self.conn.core.QueryTree(self.default_screen.root.wid).reply()
+        return list(q.children)
+
```

Comment:
We already have a similar `query_tree` method on the Window class, used by the root window's instance of that class. Would it be feasible to adapt that one for your purpose, or your version here for the other one's purpose? Otherwise there are two apparent ways of doing what appears to be the same thing.

Reply:
Yeah, I wasn't sure about this. The other method wraps the window ids with newly created windows, which seems useless overhead for the purposes of this PR. If that wasn't the case, one could just call the existing method on the root window and we wouldn't need a second implementation.

The original `query_tree` is only called in one place, I think (haven't looked through the tests, though). So it should be possible to find some common ground.

Reply:
I merged the functions. Next push will contain the changes.